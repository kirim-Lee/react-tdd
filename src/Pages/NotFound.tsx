import styled from 'styled-components';

const Container = styled.div`
  font-size: 20px;
`;
const NotFound = () => {
  return <Container>Not Found 😞</Container>;
};

export default NotFound;
