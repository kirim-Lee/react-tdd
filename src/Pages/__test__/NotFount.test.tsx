import { render, screen } from '@testing-library/react';
import NotFound from '../NotFound';

describe('Not Found', () => {
  it('렌더링 테스트', () => {
    const { container } = render(<NotFound />);

    const message = screen.getByText('Not Found 😞');
    expect(message).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
