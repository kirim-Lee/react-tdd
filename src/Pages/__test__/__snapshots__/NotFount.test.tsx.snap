// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found 렌더링 테스트 1`] = `
<div>
  <div
    class="sc-bcXHqe CFXPj"
  >
    Not Found 😞
  </div>
</div>
`;
